src/components/input-dropdown/input-dropdown.component.ts

```diff
@@ -162,7 +161,6 @@ export class InputDropdownComponent<T> implements ControlValueAccessor, AfterVie
     inputFocusHandler(): void {
         if (!this.dropdownOpen) {
             this.dropdownOpen = true;
-            this.dropdownOpenChange.emit(this.dropdownOpen);
```

Comment:
This is also called by a `focus` handler in the template, so I don't think this is the right fix. You might be able to move `this._focusFilter()` down a couple of lines to solve this problem.